Revise Konnect Cloud overview with latest marketing content (#4093)
Kong/docs.konghq.com: app/konnect/index.md

### Service Hub

> [Service Hub](/konnect/servicehub) makes internal APIs discoverable, 
> consumable, and resuable for internal development teams. Service Hub 
> enables the cataloging of all your services into a single system of record.
> This catalog represents the single source of truth for your organization’s 
> service inventory. By leveraging Service Hub, your application developers 

Reviewer: ```suggestion
[Service Hub](/konnect/servicehub) makes internal APIs discoverable, 
consumable, and reusable for internal development teams. Catalog
all your services through the Service Hub to create a single source of 
truth for your organization’s service inventory. By leveraging Service Hub, 
your application developers 
```
Combined the two "catalog" sentences to avoid repeating "single system of record" and "single source of truth", since they mean the same thing.

consumable, and reusable for internal development teams. Catalog
all your services through the Service Hub to create a single source of 
truth for your organization’s service inventory. By leveraging Service Hub, 
your application developers 
can search, discover, and consume existing services to accelerate their 
time-to-market, while enabling a more consistent end-user experience
across the organization’s applications.

[Start cataloging services with Service Hub &rarr;](/konnect/servicehub)

### Runtime Manager

[Runtime Manager](/konnect/runtime-manager) empowers your teams to securely
collaborate and manage their own set of runtimes and services without 
the risk of impacting other teams and projects. Runtime Manager instantly
provisions hosted {{site.base_gateway}} control planes and supports securely
attaching {{site.base_gateway}} data planes from your cloud or hybrid environments.

Through the Runtime Manager, increase the security of your APIs with out-of-the-box enterprise and community plugins, including OpenID Connect, Open Policy Agent, Mutual TLS, and more.

[Learn more about the Runtime Manager &rarr;](/konnect/runtime-manager)

> [Check out Konnect-compatible plugins &rarr;](/konnect/hub)

Reviewer: ```suggestion
[Check out {{site.konnect_short_name}}-compatible plugins &rarr;](/konnect/hub)
```
My bad on this one.
